packages/es-components/src/components/controls/range-selector/RangeSelector.js

```diff
@@ -0,0 +1,180 @@
+  width: 100%;
+  background: none;
+  pointer-events: none;
+  -webkit-appearance: none;
+  &:focus {
+    outline: none;
+  }
+  &::-webkit-slider-thumb {
+    height: 17px;
+    width: 17px;
+    border-radius: 50%;
+    pointer-events: auto;
+    -webkit-appearance: none;
+    background: ${props => props.thumbColor};
+  }
+  &::-moz-range-thumb {
+    height: 17px;
+    width: 17px;
+    border: none;
+    pointer-events: auto;
+    -moz-appearance: none;
+    background: ${props => props.thumbColor};
+  }
+`;
+
+function RangeSelector({
+  currentMinValue,
+  currentMaxValue,
+  minValue,
+  maxValue,
+  progressColor,
+  sliderColor,
+  thumbColor,
+  onChange,
+  ...RangeSelectorProps
+}) {
+  const initStateMin = () => {
+    if (currentMinValue < minValue) return 0;
+    if (currentMinValue > maxValue) return 100;
+    return ((currentMinValue - minValue) / (maxValue - minValue)) * 100;
+  };
+  const initStateMax = () => {
+    if (currentMaxValue > maxValue) return 100;
+    if (currentMaxValue < minValue) return 0;
+    return ((currentMaxValue - minValue) / (maxValue - minValue)) * 100;
+  };
+  const [min, setMin] = useState(currentMinValue);
+  const [max, setMax] = useState(currentMaxValue);
+  const [minPercentage, setMinPercentage] = useState(initStateMin);
+  const [maxPercentage, setMaxPercentage] = useState(initStateMax);
+
+  useEffect(() => {
+    setMin(currentMinValue);
+    setMinPercentage(initStateMin());
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [currentMinValue, minValue, maxValue]);
+
+  useEffect(() => {
+    setMax(currentMaxValue);
+    setMaxPercentage(initStateMax());
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [currentMaxValue, minValue, maxValue]);
+
+  const changeMin = event => {
+    setMin(Number(event.target.value));
+    setMinPercentage(((event.target.value - minValue) / (maxValue - minValue)) * 100);
+  };
+  const changeMinEnd = event => {
+    setMin(Number(event.target.value));
+    setMinPercentage(((event.target.value - minValue) / (maxValue - minValue)) * 100);
+    onChange(Number(event.target.value), max);
+  };
+  const changeMax = event => {
+    setMax(Number(event.target.value));
+    setMaxPercentage(((event.target.value - minValue) / (maxValue - minValue)) * 100);
+  };
+  const changeMaxEnd = event => {
+    setMax(Number(event.target.value));
+    setMaxPercentage(((event.target.value - minValue) / (maxValue - minValue)) * 100);
+    onChange(min, Number(event.target.value));
+  };
+
+  return (
+    <RangeSelectorBase {...RangeSelectorProps}>
+      <RangeSelectorSlider sliderColor={sliderColor}>
+        <RangeSelectorProgress
+          minPercentage={minPercentage}
+          maxPercentage={maxPercentage}
+          progressColor={progressColor}
+        ></RangeSelectorProgress>
+      </RangeSelectorSlider>
+      <RangeSelectorInputContainer>
+        <RangeSelectorInput
+          type="range"
+          aria-label="min-input"
```

Comment:
`aria-label` values should be in plain English, as they'll often be read aloud by Assistive Technology.
```suggestion
          aria-label="Minimum Value Input"
```

Reply:
change label to  Minimum Value Input